src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,71 @@
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import classNames from 'classnames';
+import { PersonConnection } from '../PersonConnection/Person.Connection';
+
+type PeopleTableProps = {
+  people: Person[] | undefined;
```

Comment:
Consider making `people` a required prop by removing the `undefined` type. This will simplify the logic in the component, as you won't need to use optional chaining (`?.`) when accessing `people`.